### DISCUSSION TITLE
Show current time, set timezone and NTP server in WebUI

### DISCUSSION THREAD

**Discussion by bluepuma77** [arendst/Tasmota#11768] [Ideas]
Tasmota has a nice timer menu, but the current time is completely missing in the WebUI.

1. Main menu - no current time
2. Information menu - no current time
3. Configure Timer Menu - no current time

That usually doesn't matter - until daylight saving time changes ;-)

It would be great to be able to use the WebUI to:
1. See the current time
2. Set the current timezone (like "CET")
3. Pick a NTP server
4. Optionally: set the current time manually

Using AWP07L Module with Tasmota 9.3.1

**Comment by ascillato** [comment]
You can see the current time in the console where you can also set NTP, time, timezone, etc.

**Reply by sfromis** [reply]
There is no attempt of the web UI containing more than a small subset of available options.

**Reply by AlfredoCubitos** [reply]
I'm missing that feature as well.
I think, this is an important especially for not advanced user.
For me only showing date, time and timezone in the Main or Info page would be enough. 
Please think about if you can't implement this. It would improve the usability a lot and will avoid false timer settings.

**Reply by sfromis** [reply]
Feature has been implemented some time ago, see [WebTime](https://tasmota.github.io/docs/Commands/#webtime).

**Reply by AlfredoCubitos** [reply]
aah, thank you. :-)
Is there a format option for the output available?
As I understand, I can only modify the output length with `<start_pos>,<end_pos>` , or?

**Reply by sfromis** [reply]
The format is documented in the link I posted, and you can indeed simply take a piece of that, like if you only want time.
